docs/UserGuide.md

```diff
@@ -54,6 +54,86 @@ These are symbols we will be using to highlight certain information to you.
 | :bulb:               | Tips to optimise the use of Foster Family |
 | :exclamation:        | Warning about the usage of commands       |
 
+--------------------------------------------------------------------------------------------------------------------
+## User Interface
+There are mainly two different screens that you may face using Foster Family. 
+
+### The Main View 
+![Ui](images/Ui.png)
+This is the main view that welcomes you when you first open up Foster Family.
+
+### The Profile View 
+![Profile](images/screenshots/ProfilePage.png)
+This is the profile view that you can use to add a fosterer or edit an already existing fosterer. 
+
+#### How to open profile
+There are mainly two ways you can use to open this profile page. 
+1. Enter <code>add</code> to view an empty profile page to [add](#adding-a-fosterer-through-the-profile-page-add) a fosterer.
+2. Enter either <code>edit INDEX</code> or <code>view INDEX</code> to [edit](#editing-a-fosterers-detail-through-the-profile-page-edit) or [view](#viewing-a-fosterers-detail-view) the fosterer at index <code>INDEX</code> in Foster Family.
+
+Let's say you want to open a profile of a fosterer name Benson Meie who is currently on index 1. 
+![Benson](images/screenshots/BensonMeieView.png)
+
+Simply enter <code>view 1</code> like the example above, which opens a profile page of a fosterer at index 1, Benson Meie. 
+
+![Example profile](images/screenshots/EditExample.png)
+
+#### Navigating through fields
+Typing in the name of the field of the fosterer - or letters that are close to the name of the field - directs your cursor to the textbox, allowing you to make changes.
+
+![Before typing name](images/screenshots/BeforeEnteringName.png)
+
+In the example above, entering <code>name</code>, or others that are close to name like <code>nam</code> sets focus on the name field.
+
+![Profile navigate field](images/screenshots/NavigatingFields.png)
+
+After you finish editing, pressing enter key brings your cursor back to the command box.
```

Comment:
Also, do we want to talk about esc behaviour? You talked about it for the exit without saving confirmation for the overall Person edit/add, but not wrt editing individual fields.

Reply:
Oh right forgot! I'll add it right away. Thanks!